src/plugins/ncr/sagas/index.js

```diff
@@ -2,7 +2,7 @@ import { all, fork, takeLatest } from 'redux-saga/effects';
 import { actionTypes } from '../constants';
 
 import batchOperationFlow from './batchOperationFlow';
-import cloneNodeFlow from './cloneNodeFlow';
+import cloneNodeFlow from '../../curator/sagas/cloneNodeFlow';
```

Comment:
@tbone1020 fully move this over to curator, so instead of importing like this (here in ncr) you would import it in curator/sagas. all of the related code should move too like the action and whatnot